release-notes/6.0/preview/6.0.0-preview.4-runtime-deps.json

```diff
@@ -0,0 +1,1035 @@
+            {
+              "name": "libldap.dylib",
+              "dependencyType": "Library",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.Drawing.Common",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "Carbon Framework",
+              "dependencyType": "MacOSFramework",
+              "usage": "default"
+            },
+            {
+              "name": "libgdiplus.dylib:6.0.1",
+              "dependencyType": "Library",
+              "usage": "default"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "rid": "rhel",
+      "components": [
+        {
+          "name": "Microsoft.NETCore.App",
+          "type": "Framework",
+          "platformDependencies": [
+            {
+              "name": "glibc",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "krb5-libs",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libicu",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libgcc",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libstdc++",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "lttng-ust",
+              "dependencyType": "LinuxPackage",
+              "usage": "diagnostics"
+            },
+            {
+              "name": "openssl-libs",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "tzdata",
+              "dependencyType": "LinuxPackage",
+              "usage": "localization"
+            },
+            {
+              "name": "zlib",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.DirectoryServices.Protocols",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "openldap",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            }
+          ]
+        },
+        {
+          "name": "System.Drawing.Common",
+          "type": "NuGetPackage",
+          "platformDependencies": [
+            {
+              "name": "cups-libs",
+              "dependencyType": "LinuxPackage",
+              "usage": "printing"
+            },
+            {
+              "name": "libgdiplus:6.0.1",
+              "dependencyType": "LinuxPackage",
+              "usage": "default"
+            },
+            {
+              "name": "libx11-dev",
```

Comment:
This can't be right for RHEL. RHEL packages use `-devel` suffix. This package is named `libX11-devel` on RHEL.

Reply:
Yep, I've now updated to use `libX11-devel` for RHEL, Fedora, openSUSE, and SLES.